core/src/main/kotlin/no/strooped/StroopedController.kt

```diff
@@ -31,9 +31,11 @@ class StroopedController : Game() {
             changeScreen(TaskScreen::class.java)
         }
         gameLifecycleService.onRoundEnd {
+            GameSingleton.taskNumber = 0
             GameSingleton.room?.player?.apply {
                 placement = it // new placement
             }
+            screens.remove(EndRoundScreen::class.java)
```

Comment:
I'm considering that we should just remove the `screens` Map, since we are really not taking advantage of it, and instead call `setScreen` directly. Then we can remove a lot of boilerplate code, such as `screens.put`, `changeScreen` and `screens.remove`

Reply:
yea, sure